diff --git a/src/focusblurparam.h b/src/focusblurparam.h
index 484fd7d..ed59035 100644
--- a/src/focusblurparam.h
+++ b/src/focusblurparam.h
@@ -81,7 +81,7 @@ struct _FblurParam
 
 /*---- Variable ----*/
 
-const FblurStoreParam fblur_init_param;
+extern const FblurStoreParam fblur_init_param;
 
 
 /*---- Functions ----*/
diff --git a/src/focusblurrc.h b/src/focusblurrc.h
index bc09a02..72235bc 100644
--- a/src/focusblurrc.h
+++ b/src/focusblurrc.h
@@ -40,7 +40,7 @@ struct _FblurPreferences
 
 /*---- Variables ----*/
 
-const FblurPreferences fblur_init_preferences;
+extern const FblurPreferences fblur_init_preferences;
 
 
 /*---- Functions ----*/
